CHANGELOG.md

```diff
@@ -0,0 +1,106 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
The changelog should be updated before do the merge and release it to production, do we want to do it here or for the merge to `master`?

Reply:
For the merge to `master`, in the respective release-branch.